app/models/like.rb

```diff
@@ -0,0 +1,9 @@
+class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  def update_counter
+    count = post.likes_counter || 0
+    post.update_columns(likes_counter: count + 1)
+  end
```

Comment:
- [ ] [OPTIONAL] Kindly use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - after_save to call the update_counter method every time a new post has been saved to the database, then make the update_post_counter method private